docs/machineset-aws.md

```diff
@@ -1,9 +1,14 @@
 # Creating an AWS Windows MachineSet
 
-_\<windows_container_ami\>_ can be used as it is without any modification.
-You must use Windows Server 2019 with a version 10.0.17763.1457 or earlier to work
-around Windows containers behind a Kubernetes load balancer becoming unreachable
-[issue](https://github.com/microsoft/Windows-Containers/issues/78).
```

Comment:
is this guidance no longer relevant?

Reply:
I think it should be like 'you can use 2019 or 2022` as a suggestion

Reply:
I don't think we should duplicate the [Supported Windows Server versions](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/wmco-prerequisites.md#supported-windows-server-versions). I will reword this to include a reference.